Flow.Launcher.Infrastructure/Alphabet.cs

```diff
@@ -19,160 +19,45 @@ public interface IAlphabet
-                
-                lock(_pinyinStorage)
-                {
-                    var loaded = _pinyinStorage.TryLoad(new Dictionary<string, string[][]>());
-
-                    PinyinCache = new ConcurrentDictionary<string, string[][]>(loaded);
-                }
-
-                // force pinyin library static constructor initialize
-                PinyinHelper.toHanyuPinyinStringArray('T', Format);
-            });
-            Log.Info($"|Flow Launcher.Infrastructure.Alphabet.Initialize|Number of preload pinyin combination<{PinyinCache.Count}>");
-        }
-
-        public string Translate(string str)
-        {
-            return ConvertChineseCharactersToPinyin(str);
-        }
-
-        public string ConvertChineseCharactersToPinyin(string source)
-        {
-            if (!_settings.ShouldUsePinyin)
-                return source;
-
-            if (string.IsNullOrEmpty(source))
-                return source;
-
-            if (!ContainsChinese(source))
-                return source;
-                
-            var combination = PinyinCombination(source);
-            
-            var pinyinArray=combination.Select(x => string.Join("", x));
-            var acronymArray = combination.Select(Acronym).Distinct();
-
-            var joinedSingleStringCombination = new StringBuilder();
-            var all = acronymArray.Concat(pinyinArray);
-            all.ToList().ForEach(x => joinedSingleStringCombination.Append(x));
-
-            return joinedSingleStringCombination.ToString();
-        }
-
-        public void Save()
-        {
-            if (!_settings.ShouldUsePinyin)
-            {
-                return; 
-            }
-
-            lock(_pinyinStorage)
-            {
-                _pinyinStorage.Save(PinyinCache.ToDictionary(i => i.Key, i => i.Value));
-            }            
         }
 
-        private static string[] EmptyStringArray = new string[0];
-        private static string[][] Empty2DStringArray = new string[0][];
 
-        /// <summmary>
-        /// replace chinese character with pinyin, non chinese character won't be modified
-        /// Because we don't have words dictionary, so we can only return all possiblie pinyin combination
-        /// e.g. 音乐 will return yinyue and yinle
-        /// <param name="characters"> should be word or sentence, instead of single character. e.g. 微软 </param>
-        /// </summmary>
-        public string[][] PinyinCombination(string characters)
+        public string Translate(string content)
         {
-            if (!_settings.ShouldUsePinyin || string.IsNullOrEmpty(characters))
-            {
-                return Empty2DStringArray;
-            }
-
-            if (!PinyinCache.ContainsKey(characters))
+            if (_settings.ShouldUsePinyin)
             {
-                var allPinyins = new List<string[]>();
-                foreach (var c in characters)
+                if (_pinyinCache.ContainsKey(content))
```

Comment:
I think this should be !_pinyinCache.ContainsKey(content) instead

Reply:
Oh that's true, such a silly mistake 😅, thank you for indicating the error! I will change it later.